nix/README.md

```diff
@@ -22,3 +22,25 @@ make shell TARGET_OS=android
 This way your shell and all other nix commands should run in a setup that is tailored towards Android development.
 
 For valid values you can check the [`nix/platform.nix`](/nix/platform.nix) file.
+
+# Known Issues
+
+## MacOS 10.15 "Catalina"
+
+There is a unsolved issue with the root(`/`) file system in `10.15` being read-only:
+https://github.com/NixOS/nix/issues/2925
+
+Our current recommended workaround is putting `/nix` under `/opt/nix` and symlinking it via `/etc/synthetic.conf`:
+```
+sudo mkdir /opt/nix
+sudo chown ${USER} /opt/nix
+# the divider here needs to be a tab, otherwise /etc/synthetic.conf ignores it
+sudo sh -c "echo 'nix	/opt/nix' >> /etc/synthetic.conf"
```

Comment:
Doesn't this work? If so, we could avoid the warning above.
```suggestion
sudo sh -c "echo 'nix\t/opt/nix' >> /etc/synthetic.conf"
```

Reply:
Oh, it does:
```
administrator@macos-01.ms-eu-dublin.ci.misc:~ % echo 'x\tw'
x	w
```
On linux that requires `-t`.